lesson_07/homework_07.py

```diff
@@ -0,0 +1,158 @@
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+
+
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while multiplier <= 5:
+        result = number * multiplier
+        # десь тут помила, а може не одна
+        if result > 25:
+            # Enter the action to take if the result is greater than 25
+            break
+        print(str(number) + "x" + str(multiplier) + "=" + str(result))
+
+        # Increment the appropriate variable
+        multiplier += 1
+
+
+multiplication_table(3)
+# Should print:
+# 3x1=3
+# 3x2=6
+# 3x3=9
+# 3x4=12
+# 3x5=15
+
+
+# task 2
+"""  Написати функцію, яка обчислює суму двох чисел.
+"""
+
+
+def sum_two_integers(first_integer: int, second_integer: int):
+    return first_integer + second_integer
+
+
+# task 3
+"""  Написати функцію, яка розрахує середнє арифметичне списку чисел.
+"""
+
+
+def avarage_of_integers(*args: int):
+    return sum(args) / len(args)
+
+
+# task 4
+"""  Написати функцію, яка приймає рядок та повертає його у зворотному порядку.
+"""
+
+
+def revert_string(my_string: str):
```

Comment:
We have builtin function to reverting our string. You function can be in one line with return of reversed(my_string). Please try to use more flexible solutions in your code)